PWGCF/EbyEFluctuations/Tasks/CMakeLists.txt

```diff
@@ -19,3 +19,9 @@ o2physics_add_dpl_workflow(mean-pt-fluc-id
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2Physics::PWGCFCore
                     COMPONENT_NAME Analysis)
 
+
+o2physics_add_dpl_workflow(mean-pt-pbpb-fluc
```

Comment:
Please, don't use the system (`pbpb`) in the name
At some point the system on which the analysis might run will be a configurable, i.e. the analysis will be able to run on `pp` or `PbPb` just changing a configurable parameter

As this is an identified particle analysis please make that explicit in the executable name and also in the name of the source file

Reply:
If you push your new commits to your gitHub repo the PR will get automatically updated

Reply:
Thank you Victor for the comments. I will modify the task according to your suggestions. 
After modifying the comments do i need to create a new pull request again with the modify code and CMakeList.txt?

> If you push your new commits to your gitHub repo the PR will get automatically updated

Thanks Victor. I will push new commits by today.

Reply:
> If you push your new commits to your gitHub repo the PR will get automatically updated 

Hii Victor,
I have push my new commit in the branch MeanpTPID. I am getting some error in MegaLinter. Could you please guide me what could be the issue?
Thanks in Advance!
Regards
Sweta